packages/tof_driver/include/hardware_test_tof/hardware_test_tof.py

```diff
@@ -0,0 +1,58 @@
+import rospy
+
+from dt_duckiebot_hardware_tests import HardwareTest, HardwareTestJsonParamType
+
+from tof_accuracy import ToFAccuracy
+
+
+class HardwareTestToF(HardwareTest):
+    def __init__(self, sensor_name: str, accuracy: ToFAccuracy) -> None:
+        super().__init__()
+        # attr
+        self._sensor_name = sensor_name  # e.g. "front_center"
+        self._sensor_name_human = sensor_name.replace("_", " ")  # e.g. "front center"
+        self._accuracy = accuracy
+
+    def test_id(self) -> str:
+        return f"Time-of-Flight ({self._sensor_name})"
+
+    def test_description_preparation(self) -> str:
+        return self.html_util_ul(
+            [
+                "Put your Duckiebot in its normal orientation.",
+                "Place it near you, with its camera facing an empty space.",
+            ]
```

Comment:
Place your Duckiebot on a flat surface with at least one meter of empty space in front of the Time-of-Flight sensor on the front bumper.